js/map_utils.js

```diff
@@ -170,10 +170,6 @@ function updateCategory(category, prevCategory, action) {
   
   updatePies(category, prevCategory);
   
-  updateLegend(category);
-  if(updateCategoryTimer){
-    clearTimeout(updateCategoryTimer);
-  }
```

Comment:
I think these three lines 
```
if(updateCategoryTimer){
  clearTimeout(updateCategoryTimer);
}
```
are relevant to google analytics rather than to the legend, aren't they? and so shouldn't be deleted? or am i misunderstanding their function? @wdwatkins ?

Reply:
oh I see now, my bad.